packages/mui-material/src/Breadcrumbs/Breadcrumbs.d.ts

```diff
@@ -14,6 +14,41 @@ export interface BreadcrumbsTypeMap<P = {}, D extends React.ElementType = 'nav'>
      * Override or extend the styles applied to the component.
      */
     classes?: Partial<BreadcrumbsClasses>;
+    /**
+     * The components used for Collapsed item type
+     * @default {
+     *   Collapsed: MoreHorizIcon,
+     * }
+     */
+    components?: {
+      Collapsed?: React.ElementType;
+    };
+    /**
+     * The props used for each slot inside.
+     * @default {}
+     */
+    componentsProps?: {
```

Comment:
The keys in the `componentsProps` should be the same as the keys in `components` (just lowercased).